frontend/appflowy_flutter/pubspec.yaml

```
@@ -15,7 +15,7 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 1.0.0+1
+version: 0.1.1+1
```

```suggestion
version: 0.1.1
```
You may omit the build number.
> https://dart.dev/tools/pub/pubspec#version